src/daemon/mod.rs

```diff
@@ -202,10 +204,22 @@ async fn maybe_start_metrics_service(
         );
         let db_directory = crate::db::db_engine::db_root(&chain_path(config))?;
         let db = ctx.db.writer().clone();
+        let chain_store = ctx.state_manager.chain_store().clone();
+        let get_chain_head_height = move || chain_store.heaviest_tipset().epoch();
+        let chain_store = ctx.state_manager.chain_store().clone();
+        let get_chain_head_network_version = move || {
+            let epoch = chain_store.heaviest_tipset().epoch();
+            chain_store.chain_config.network_version(epoch)
+        };
```

Comment:
Do you think it resolves the issue we had with calibnet nodes not updated? https://github.com/ChainSafe/forest/issues/5141#issuecomment-3291159452

Reply:
Let me rephrase; you're checking what the network is _supposed_ to be according to the schedule. What is more useful for monitoring the health of the nodes, is checking the _actual_ network version (though the expected version is also good to have, so consumers can set up an alarm if there's a mismatch).

Reply:
@LesnyRumcajs  I think with `network_version_revision`, the issue could be resolved. Please let me know your thoughts